packages/ember-data/lib/serializers/json-serializer.js

```diff
@@ -603,9 +603,7 @@ var JSONSerializer = Serializer.extend({
         if (relationshipMeta.kind === 'belongsTo') {
           data = this.extractRelationship(relationshipMeta.type, relationshipHash);
         } else if (relationshipMeta.kind === 'hasMany') {
-          data = Ember.A(relationshipHash).map(function(item) {
```

Comment:
`Ember.A` was removed during the cleaning. Was it intentional or by mistake?

Indeed, now it breaks when you push a record with hasMany relationships which are not defined in the payload. The workaround is to set them as an empty array but it complicates unnecessarily the payload.

Example with this model:
```javascript
// models/list.js
export default DS.Model.extend({
  name: attr('string'),
  items: hasMany('item'),
});
```
In ED 1.13:
```javascript
store.push({ type: 'list', id: 1, attributes: { name: 'main' });
```
In ED 2.0:
```javascript
store.push({ type: 'list', id: 1, attributes: { name: 'main' }, relationships: { items: { data: [] }}});
```
Is it the new recommended way to set the hasMany relationship payload or a mistake during this cleaning?